modules/reference/pages/rpk/rpk-transform/rpk-transform-list.adoc

```diff
@@ -0,0 +1,39 @@
+= rpk transform list
+:description: rpk transform list
+
+List data transforms
+
+== Usage
+
+[,bash]
+----
+rpk transform list [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+list, ls
+----
+
+== Flags
+
+[cols="1m,1a,2a]
+|===
+|*Value* |*Type* |*Description*
+
+|`-d, --detailed` |- |Print per-partition information for data transforms.
```

Comment:
@rockwotj what does this actually mean? partitions the transform is avctively running on?  wouldn't that always be all partitions in the topic.  What information does this show?

Reply:
It is the per transform lag and the status of each partition (error, inactive, running, etc)

Reply:
`rpk transform` edits are in https://github.com/redpanda-data/docs/pull/104